### ISSUE TITLE
Cannot add new git+https repo with helm v3.0.0 and helm-git v0.4.2

### ISSUE THREAD

**Issue by mojochao** [aslafy-z/helm-git#49]
With helm v2.16.1 I am able to run the following:

```
$ helm repo add coresvcs-charts `git+https://bitbucket.org/sambatv/coresvcs-charts@?ref=master'
```

With helm v3.0.0 I get an error:

```
$ helm repo add coresvcs-charts 'git+https://bitbucket.org/sambatv/coresvcs-charts@?ref=master'
Error: unknown flag: --home
helm.go:76: [debug] unknown flag: --home
Error: looks like "git+https://bitbucket.org/sambatv/coresvcs-charts@?ref=master" is not a valid chart repository or cannot be reached: plugin "helm-git" exited with error
```

Does the helm-git plugin support Helm 3.0.0?

Many thanks in advance.

**Comment by mojochao** [comment]
Apparently, this plugin relies on features removed from helm v3.0.0.

https://github.com/aslafy-z/helm-git/blob/master/helm-git-plugin.sh#L81

**Comment by aslafy-z** [comment]
This plugin does not support Helm 3 from now. I'd be happy to accept PR regarding this as I'm not using it from now.
Regards,

**Comment by iAnomaly** [comment]
@mojochao @aslafy-z: #50

**Comment by aslafy-z** [comment]
I just released [`v0.5.0`](https://github.com/aslafy-z/helm-git/releases/tag/v0.5.0). You can have install it by running:
```shell
$ helm plugin install https://github.com/aslafy-z/helm-git.git --version 0.5.0
```

Thanks for reporting @mojochao!
Thanks for the quick fix @iAnomaly!
